.github/actions/setup-sentry/action.yml

```diff
@@ -126,4 +131,8 @@ runs:
           sentry devservices up clickhouse snuba
         fi
 
+        if [ "$NEED_CHARTCUTERIE" = "true" ]; then
+          sentry devservices up --skip-only-if chartcuterie
```

Comment:
For anyone coming after me, it does start:
<img width="547" alt="image" src="https://user-images.githubusercontent.com/44410/112210262-d9f42480-8bf0-11eb-8a88-e30f1bb03113.png">